docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
yes, it's exposed for the components don't want to use `createI13nNode`, so far as I know, people always use createI13nNode, 

and `createI13nNode` actually makes more sense for it's naming, (`I13nMixin` doesn't provide enough information about what it does), and it let users to set `options` easily https://github.com/yahoo/react-i13n/blob/master/docs/api/createI13nNode.md#createi13nnodecomponent-options,